app/models/bgs_dependents/adult_child_attending_school.rb

```diff
@@ -64,7 +66,8 @@ def described_class_attribute_hash
       {
         ssn: @ssn,
         birth_date: @birth_date,
-        ever_married_ind: @was_married == true ? 'Y' : 'N'
+        ever_married_ind: @was_married == true ? 'Y' : 'N',
```

Comment:
Can you use the formatted_boolean method here for the ever_married_ind value logic?

Reply:
Great observation Lindsey!  The ever_married_ind was there prior to the formatted_boolean method being created.  I've updated this and ran a few spec tests just to make sure it is working properly. Thank you for the suggestion.